python/tskit/trees.py

```diff
@@ -6432,14 +6432,9 @@ def write_vcf(
         with sample nodes are included in increasing order of individual ID.
 
         Subsets or permutations of the sample individuals may be specified
-        using the ``individuals`` argument. It is an error to specify any
-        individuals that are not associated with any nodes, or whose
-        nodes are not all samples.
-
-        Mixed-sample individuals (e.g., those associated with one node
-        that is a sample and another that is not) in the data model will
-        result in an error by default. However, such individuals can be
-        excluded using the ``individuals`` argument.
+        using the ``individuals`` argument. An individual specified that
+        is not associated with any nodes will be ignored and not included in the
```

Comment:
That's not true any more

Reply:
I thought it was - I added a test in 266f0cb2774abc2c3f86c02c978fb74d8e022c31 to check.